pyproject.toml

```diff
@@ -20,3 +20,6 @@ black = "^20.8b1"
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.pyright]
+exclude = [".env", "env.example"]
```

Comment:
Are we using pyright?

Reply:
Pyright is apparently used by VS Code. However, the problem with the `.env` file doesn't appear to be related to pyright misbehaving so this change doesn't seem to help.